app/lib/fieldoptions.php

```diff
@@ -58,6 +58,10 @@ public function __construct($field) {
           $path = str::substr($path, 3);
         }
         $page = $currentPage;
+
+        if ($path && $currentPage->find($path)) {
```

Comment:
We're testing to see if there are any remaining characters in the passed `$path` after popping off the leading `../`s, then making sure we can locate a child of `$currentPage`, before doing the assignment.